pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -120,6 +121,17 @@ func (trs *TaskRunStatus) MarkResourceNotConvertible(err *CannotConvertError) {
 	})
 }
 
+// MarkResourceFailed sets the ConditionSucceeded condition to ConditionFalse
+// based on an error that occurred and a reason
+func (trs *TaskRunStatus) MarkResourceFailed(reason string, err error) {
```

Comment:
Not sure if this should be in alpha too...

Reply:
```
// TaskRunStatus defines the observed state of TaskRun
type TaskRunStatus = v1beta1.TaskRunStatus
```

It doesn't as it's just an alias in `v1alpha1` :wink: